edge/cmd/edgecore/app/server.go

```diff
@@ -144,6 +146,20 @@ func environmentCheck() error {
 	return nil
 }
 
+// get edge node local ip
+func GetNodeIp(config *v1alpha1.EdgeCoreConfig) *v1alpha1.EdgeCoreConfig {
```

Comment:
Is it need export?

Reply:
This method has not been called elsewhere, feels it is no need to export currently.

Reply:
If so, no need this func. Just write as:
```
if config.Modules.Edged.NodeIP == "" {
  ...
  config.Modules.Edged.NodeIP = localIP
}
```

Reply:
OK.